### ISSUE TITLE
Laggy and Account gets disconnected

### ISSUE THREAD

**Issue by prateekch33** [microsoft/devhome#985]
### Dev Home version

0.137.141.0

### Windows build number

10.0.23466.1001

### Other software

OS Build Version: 10.0.23466.1001.amd64fre.ni_prerelease.230522-1422
.NET Version: .NET 6.0.16

### Steps to reproduce the bug

Settings -> Add Account -> Login Prompt on Browser -> After Loging Error Shows

On reopening Linked Account got unlinked.

### Expected result

No error will be shown on successful sign in and on reopening account should be linked unless I unlink that.

### Actual result

Error Shows and Account getting unlinked.

### Included System Information

CPU: AMD Ryzen 5 3500U with Radeon Vega Mobile Gfx  
Physical Memory: 13.92GB (6.89GB free)
Processor Architecture: x64

### Included Extensions Information

Extensions: 
Microsoft.Windows.DevHome_0.137.141.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHomeGitHubExtension_0.137.141.0_x64__8wekyb3d8bbwe

**Comment by mikebattista** [comment]
Could you clarify what error you got? Are you saying despite the error the account was connected, and then it got unlinked?

**Comment by prateekch33** [comment]
Yes, the prompt on Dev Home shows Something Went Wrong, but still my account got linked. But when I close the dev home and re open it the account got unlinked.

**Comment by mikebattista** [comment]
Ok. Thanks. We'll have to investigate.

**Comment by prateekch33** [comment]
After the new Update, this issue has been resolved thanks for your help.
